### PR TITLE
update SceneTree docs to include note about change_scene_to with null…

### PR DESCRIPTION
**Pull request by twobitadder**
This aims to be (the smaller) half of the fix for #63565 - I am happy to take a stab at the proposed edits to `master`, but this was low-hanging fruit that I thought could be pretty easy to sneak in. Feel free to provide input on the wording, I'm sure it could be made clearer. Let me know too if there's anything else I should include!

### CODE REVIEW

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Note: This would be good to cherrypick to `master` too.

File: doc/classes/SceneTree.xml [hunk 1]
```diff
@@ -55,6 +55,7 @@
 				Changes the running scene to a new instance of the given [PackedScene].
 				Returns [constant OK] on success or [constant ERR_CANT_CREATE] if the scene cannot be instantiated.
 				[b]Note:[/b] The scene change is deferred, which means that the new scene node is added on the next idle frame. You won't be able to access it immediately after the [method change_scene_to] call.
+				[b]Note:[/b] Passing a value of [code]null[/code] into the method will unload the current scene without loading a new one. Additionally, passing in any [Object] type that is not a [PackedScene] will be treated as passing [code]null[/code].
```
Comment (akien-mga) [inline doc/classes/SceneTree.xml:58]: It's good to mention what null does, but I wouldn't add the caveat for passing an invalid Object. That's a GDScript implementation detail and is true for every method in the codebase.

Comment (twobitadder) [reply doc/classes/SceneTree.xml:58]: That's a fair point! Removed.

**Comment by akien-mga** [comment]
Looks good! Could you squash the commits? See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase) for instructions.

**Comment by twobitadder** [comment]
Well 29 pending reviewers - THAT doesn't seem right. I think maybe I did something wrong when rebasing.

**Comment by twobitadder** [comment]
Yup, looks like I rebased the wrong branch. I'll go ahead and close this and fix it on my end.

**Comment by twobitadder** [comment]
282 commits, lol, lmao

**Pull request closed without merging**